File: Algorithms/JS/TwoPointers/remove-element.js
let nums = [1,2,2,3]
let val = 3

var removeElement = function(nums, val) {
  let k = 0
  for(let i = 0; i < nums.length; i++){
    if(nums[i] != val){
      nums[k] = nums[i]
      k++
    }
  }
  return nums
};

console.log(removeElement(nums,val))

File: Java/DSA/src/Arrays/RemoveElement.java
package Arrays;

public class RemoveElement {
    public static void main(String[] args) {
        int[] nums = {3,2,2,3};
        int val = 3;
        int k = 0;
        for (int i = 0; i < nums.length; i++) {
            if(nums[i] != val) {
                nums[k] = nums[i];
                k++;
            }
        }
        for (int x : nums) {
            System.out.println(x);
        }
    }
}
